### PR TITLE
 Refactor: Migrate message commands to aleph-sdk-python #150 

### PR DESCRIPTION
**Pull request by 1yam**
From : https://github.com/aleph-im/aleph-client/pull/150

Refactor:

- Remove deprecated example of usage using mqtt.py.
- Fix mypy errors:
  - Replace typer.echo with typer.secho in account module.
  - Resolve type conversion errors in message module.
- Fix unit tests and integration tests.

### CODE REVIEW

File: tests/integration/itest_posts.py [hunk 1]
```diff
@@ -1,61 +1,56 @@
...
-        get_messages,
-        response_contains_messages,
-        timeout=5,
-        hashes=[post_message.item_hash],
-        api_server=receiver_node,
+    data = {'content': 'test'}
+    async with AuthenticatedAlephClient(
+            account=fixture_account, api_server=sdk_settings.API_HOST
+    ) as client:
+        result, status = await client.create_post(
```
Comment (odesenfans) [inline tests/integration/itest_posts.py:30]: Rename result to message.

Comment (odesenfans) [reply tests/integration/itest_posts.py:30]: Also, now that the SDK returns a status, you can assert that the operation worked with `assert status == ...`.

File: tests/integration/itest_posts.py [hunk 2]
```diff
@@ -1,61 +1,56 @@
...
+            post_type="POST",
+            ref=None,
+            channel="INTEGRATION_TESTS",
+            inline=True,
+        )
+
+    response = await try_until(
+        get_message,
+        lambda r: r is not None and r.content is not None,
+        timeout=50,
```
Comment (odesenfans) [inline tests/integration/itest_posts.py:41]: Why 50? Revert to 5, that's too long. Note that you can also specify `sync=True` in `client.create_post` to make sure that the message was processed on the API node.

**Review by odesenfans — CHANGES_REQUESTED** [review]

File: src/aleph_client/commands/message.py [hunk 3]
```diff
@@ -52,40 +63,45 @@ def find(
...
+    content_types_s: Optional[list[str]] = None
+    content_keys_s: Optional[list[str]] = None
+    refs_s: Optional[list[str]] = None
+    addresses_s: Optional[list[str]] = None
+    tags_s: Optional[list[str]] = None
+    hashes_s: Optional[list[str]] = None
+    channels_s: Optional[list[str]] = None
+    chains_s: Optional[list[str]] = None
+
+    content_types_s = content_types.split(",") if content_types else None
```
[the remark this answers is not in the document]
Comment (1yam) [reply src/aleph_client/commands/message.py:76]: _s = _splited in my head

File: tests/integration/itest_posts.py [hunk 4]
```diff
@@ -1,61 +1,56 @@
...
-        return len(response.messages) > 0
-
-    # create_message = Message(**created_message_dict)
-    response_dict = await try_until(
-        get_messages,
-        response_contains_messages,
-        timeout=5,
-        hashes=[post_message.item_hash],
-        api_server=receiver_node,
+    data = {'content': 'test'}
```
[the remark this answers is not in the document]
Comment (1yam) [reply tests/integration/itest_posts.py:26]: Used here: https://github.com/1yam/aleph-client/blob/88843397a79658aa14bc4294d0f4c164a672661c/tests/integration/itest_posts.py#L31C9-L31C9

**Review by odesenfans — APPROVED** [review]

**Pull request merged**